robottelo/cli/hostcollection.py

```diff
@@ -35,25 +37,42 @@ class HostCollection(Base):
     command_base = 'host-collection'
 
     @classmethod
+    def transform_ids(cls, options=None):
+        """Workaround host unification feature not being completed to use host
+        id instead content host uuid
```

Comment:
Host unification is a new feature for Satellite and is not finished yet, so it doesn't make sense to input any BZs. That solution is permanent until all necessary actions will be taken from dev side and Content Host entity will be removed from the system entirely. We don't need github issue either as there nothing to track as once we get any necessity to revisit current code, we will get bunch failures immediately, so will just need to fix them (remove corresponding modules, do a lot of refactoring and etc)